internal/service/folders.go

```diff
@@ -262,6 +264,7 @@ func (s *DashNGoImpl) UploadFolders(filter filters.Filter) []string {
 		}
 		params := folders.NewCreateFolderParams()
 		params.Body = &newFolder
+		//f, err := s.client.Folders.CreateFolder(params, s.getAuth())
```

Comment:
This is the same as the following line unless I'm missing something.

Reply:
Removed